src/components/Molecules/Hero/index.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <header class="page-header">
+    <h1 class="page-header__header">Porzuć niewygodne narzędzia i stertę notatek!</h1>
+    <p
+      class="page-header__paragraph"
+    >Z naszym narzędziem oszczędzimy Ci masę czasu! Załóż bezpłatne konto i sprawdź jakie to łatwe.</p>
+    <div class="page-header__buttons">
+      <Button value="Sprawdź" />
+      <Button class="outline" value="Kontakt" />
+    </div>
+  </header>
+</template>
+<script>
+export default {
+  name: "Hero"
+};
+</script>
+<style lang="scss" scoped src="./style.scss">
```

Comment:
Is it possible to close style line without new line?
```<style lang="scss" scoped src="./style.scss" />```
```<style lang="scss" scoped src="./style.scss"></style>```

Reply:
#Prettier